common-lib/src/cmr/common/cache/cache_spec.clj

```diff
@@ -1,13 +1,14 @@
 (ns cmr.common.cache.cache-spec
   "Defines a common set of tests for checking cache implementations"
-  (:require [clojure.test :refer :all]
-            [cmr.common.cache :as c]
+  (:require [clojure.test :refer [are is testing]]
```

Comment:
our standards have the [clojure.test ... on the next line and lines up with : (colon)   Look at the next file as an example.

Reply:
I have been converting those when I notice them, I'll change this one.